src/status_im/ui/screens/hardwallet/setup/views.cljs

```diff
@@ -161,6 +166,48 @@
           :uppercase? false
           :forward?   true}]]])))
 
+(defview enter-recovery-phrase []
+  (letsubs [width [:dimensions/window-width]
+            recovered-account [:get-recover-account]]
+    (let [{:keys [passphrase-valid? passphrase-error passphrase-warning]} recovered-account
+          disabled? (not passphrase-valid?)]
+      [react/view styles/enter-pair-code-container
+       [react/view styles/enter-pair-code-title-container
+        [react/view
+         [react/text {:style           styles/center-title-text
+                      :number-of-lines 2
+                      :font            :bold}
```

Comment:
hm do we have this font ? i thought it should be font-weight in style

Reply:
good point, removing this line.